main/src/library/Epoch.flix

```diff
@@ -0,0 +1,23 @@
+namespace Epoch {
+
+    ///
+    /// Returns the number of nano seconds since the Unix epoch.
+    ///
+    pub def nanoSeconds(): Int64 & Impure = ???
+
+    ///
+    /// Returns the number of micro seconds since the Unix epoch.
+    ///
+    pub def microSeconds(): Int64 & Impure = ???
+
+    ///
+    /// Returns the number of mili seconds since the Unix epoch.
+    ///
+    pub def miliSeconds(): Int64 & Impure = ???
```

Comment:
should be `milliseconds` if [Wiktionary](https://en.wiktionary.org/wiki/millisecond) is to be trusted.

Reply:
same for nanoseconds etc?

Reply:
I was referring to the double `l`. But all-lowercase for all of them feels more natural to me.

Reply:
Doh,ok

Reply:
Ok changing it to lower case and double l :)